streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -266,6 +266,10 @@
     public static final String ZOOKEEPER_CONNECT_CONFIG = "zookeeper.connect";
     private static final String ZOOKEEPER_CONNECT_DOC = "Zookeeper connect string for Kafka topics management.";
 
+    /** {@code replication.factor} */
+    public static final String MIN_IN_SYNC_REPLICAS_CONFIG = "min.insync.replicas";
```

Comment:
@ijuma we currently only allow setting of `replication.factor` and now this one. Anything else will be ignored.